publish.js

```diff
@@ -0,0 +1,14 @@
+var ghpages = require('gh-pages');
+
+ghpages.publish('public', {
+  user: {
+    name: 'OpenFisca-Bot',
+  },
+  branch: 'gh-pages',
+  message: 'Auto-commit from master branch'
+}, function(err) {
+  if(err){
```

Comment:
I strongly recommend to add spaces around conditions, so that they cannot be mistaken for function calls.

`if(err){` → `if (err) {`